libs/application/templates/parental-leave/src/lib/directorateOfLabour.utils.ts

```diff
@@ -135,3 +136,60 @@ export const calculateRemainingNumberOfDays = (
 
   return availableDays - existingDays
 }
+
+/**
+ * Calculate a new date according to VMST's rules
+ */
+export const calculateDateWithNewPeriod = (start: Date, daysToUse: number) => {
+  let daysLeft = daysToUse
+  let endDate = start
+
+  while (daysLeft > 0) {
+    const currentYear = endDate.getFullYear()
+    const currentMonthIndex = endDate.getMonth()
+    const currentDayOfMonth = endDate.getDate()
+    const daysInMonth = getDaysInMonth(new Date(currentYear, currentMonthIndex))
+    const weight = daysInMonth / DAYS_IN_MONTH
+    const reverseWeight = DAYS_IN_MONTH / daysInMonth
```

Comment:
I know I'm to blame with these variable names but perhaps it would be more descriptive to call them

```typescript
const plDayToRealDayMultiplier = daysInMonth / DAYS_IN_MONTH
const realDayToPLDayMultiplier = DAYS_IN_MONTH / daysInMonth
```

Reply:
Good idea, will do that